ts/src/base/Exchange.ts

```diff
@@ -5803,7 +5803,7 @@ export default class Exchange {
                 errors = 0;
                 const responseLength = response.length;
                 if (this.verbose) {
-                    const cursorMessage = 'Cursor pagination call ' + i + 1 + ' method ' + method + ' response length ' + responseLength + ' cursor ' + cursorValue;
+                    const cursorMessage = 'Cursor pagination call ' + (i + 1) + ' method ' + method + ' response length ' + responseLength + ' cursor ' + cursorValue;
```

Comment:
Did you test this with c#?
I think just in case I would specify to convert `(i+1).toString ()` not to mix types

Reply:
No, I didn't. But I'm sure it will produce wrong JS code 💯 
moreover at all I still not sure, why need to replace 

    this.log ('Cursor pagination call', i + 1, 'method', method, 'response length', responseLength, 'cursor', cursorValue);
    
to 
    
    const cursorMessage = 'Cursor pagination call ' + i + 1 + ' method ' + method + ' response length ' + responseLength + ' cursor ' + cursorValue;
     this.log (cursorMessage);
     
 IMO first syntax is valid for any currently supported languages.
 I'm suggesting to revert to the initial code style

Reply:
@olegtaranenko We can keep the () for sure but it is safer to do ToString()

Reply:
I will update it don't worry

Reply:
love the branch's name tho 😅 
![image](https://github.com/ccxt/ccxt/assets/43336371/863d2c22-1435-474e-bc3b-eb715b5289ee)

Reply:
@carlosmiei we have 2 places with (i+1)

Reply:
@olegtaranenko 🤦 you're right, but will open a new one to fix it thanks